### ISSUE TITLE
solojuegospc.net [Anti-Adblock]

### ISSUE THREAD

**Issue by xDarkARG** [reek/anti-adblock-killer#3729]
http://www.insuranceloansonline.com/posts/cienciayeducacion/13/Semana-negra-en-la-bolsa-de-valores-de-China-y-el-mundo.html#FU2FsdGVkX1+kD1BPjTEZarJaaRGIy/udEab+ekBMbXVTf5MUjvKVG6iwCnBwK6pQvVEio7s0wxTF0iiWZHcicw==

**Comment by uBlock-user** [comment]
Can't reproduce.

**Comment by lain566** [comment]
@jspenguin2017 Can you reproduce it?

![forceads_-_google_chrome_2017-11-12_16-53-20](https://user-images.githubusercontent.com/27441315/32700788-39ef7072-c7ca-11e7-8382-9c7f4f06fe2e.png)

**Comment by jspenguin2017** [comment]
`||doopads.com/DoopADS.js` works?

**Comment by lain566** [comment]
@jspenguin2017 That does not work for me, it should look like this
<details><summary>Screenshot</summary>

![forceads_-_google_chrome_2017-11-12_17-15-05](https://user-images.githubusercontent.com/27441315/32701022-d0b2e7b0-c7ce-11e7-847f-4f69a7a37abe.png)
![5_tips_para_que_ahorres_dinero_-_insurance_loans_o_2017-11-12_17-34-42](https://user-images.githubusercontent.com/27441315/32701105-d0e7896a-c7cf-11e7-8f46-14e0e3f1b583.png)
</details>

this message appears:

`See download links`
And
`Click on the advertising and wait 30 seconds`

Similar to https://github.com/jspenguin2017/uBlockProtector/issues/472#issuecomment-322270083

**Comment by jspenguin2017** [comment]
What's the destination page? I thought it's an ad that blocks the article.

**Comment by lain566** [comment]
`http://solopaste.info/index.php?v=2304`

**Comment by jspenguin2017** [comment]
OK, I see... Tough one, it's not hard coded in DOM.

**Comment by uBlock-user** [comment]
Is the test link - http://solojuegospc.net/call-of-duty-wwii-deluxe-edition-mega-47-4-gb.html ?

**Comment by jspenguin2017** [comment]
Time to reverse this obfuscated payload... 
![image](https://user-images.githubusercontent.com/7283682/32701607-1322c96e-c796-11e7-8697-b015cec7a9dc.png)

**Comment by lain566** [comment]
@uBlock-user  This `http://solojuegospc.net/la-torre-oscura-l-lat-ing-l-1080p-l-brrip-l-x264.html`
<details>

![la_torre_oscura_l_lating_l_1080p_l_brrip_l_x264__2017-11-12_18-49-12](https://user-images.githubusercontent.com/27441315/32701683-522aa066-c7da-11e7-9fb3-25084ddbff03.png)
</details>

**Comment by jspenguin2017** [comment]
Got it.

**Comment by uBlock-user** [comment]
Yes what's wrong with that ? Is that supposed to look anything else ?
